src/plugins/intel_gpu/include/intel_gpu/primitives/dft.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2021-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "primitive.hpp"
+
+namespace cldnn {
+struct dft: public primitive_base<dft> {
+    CLDNN_DECLARE_PRIMITIVE(dft)
+
+    enum Kind {
+        forward,
+        inverse,
+    };
+
+    dft(const primitive_id &id, primitive_id &&data, std::vector<int64_t> &&axes, const layout &output_layout,
+        Kind kind, const primitive_id &ext_prim_id = { }) :
+            primitive_base { id, { move(data) }, ext_prim_id, output_layout.data_padding },
+            axes { move(axes) },
+            output_layout { output_layout },
+            kind { kind } {
+    }
+    std::vector<int64_t> axes;
+    layout output_layout;
```

Comment:
Please get rid of output_layout from primitive and let it to be set from program_node

Reply:
Done

Reply:
Its still here and seems no need?